libfreerdp/core/redirection.c

```diff
@@ -45,58 +47,119 @@ struct rdp_redirection
 	char* TargetNetAddress;
 	UINT32 TargetNetAddressesCount;
 	char** TargetNetAddresses;
+	UINT32 RedirectionGuidLength;
```

Comment:
Should all length fields be defined either with `UINT32` or with `DWORD`?

Reply:
It's pretty much a style thing, but usually we use `UINT32` for this.

Reply:
the types in `winpr` are mimicking `windows` types, so there is some ambiguity.
preferring the expressive types over the more obscure ones.